aws/resource_aws_instance_test.go

```diff
@@ -5131,6 +5132,23 @@ resource "aws_instance" "test" {
 `, rName))
 }
 
+const testAccAwsEc2InstanceConfigDynamicEBSBlockDevices = `
+resource "aws_instance" "test" {
+  ami           = "ami-55a7ea65"
```

Comment:
Avoid using hardcoded AMI IDs. Use `testAccLatestAmazonLinuxHvmEbsAmiConfig()` and `data.aws_ami.amzn-ami-minimal-hvm-ebs.id` instread.

Reply:
Will do; That's fixed over in https://github.com/terraform-providers/terraform-provider-aws/pull/14159 and I'll do the same here.

Reply:
Done.
Relevant acceptance test:

```console
$ make testacc TEST=./aws/ TESTARGS='-run=TestAccAWSInstance_EbsRootDevice_MultipleDynamicEBSBlockDevices'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 2 -run=TestAccAWSInstance_EbsRootDevice_MultipleDynamicEBSBlockDevices -timeout 120m
=== RUN   TestAccAWSInstance_EbsRootDevice_MultipleDynamicEBSBlockDevices
=== PAUSE TestAccAWSInstance_EbsRootDevice_MultipleDynamicEBSBlockDevices
=== CONT  TestAccAWSInstance_EbsRootDevice_MultipleDynamicEBSBlockDevices
--- PASS: TestAccAWSInstance_EbsRootDevice_MultipleDynamicEBSBlockDevices (211.77s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	211.825s
```